### PR TITLE
Minor code fixes in models

### PR DESCRIPTION
**Pull request by mfojtik**
- Removed unused local variables
- Removed local variables where they are not needed

### CODE REVIEW

File: src/app/models/provider_account.rb [hunk 1]
```diff
@@ -165,19 +170,17 @@ def populate_profiles_and_validate
...
     end
-    true
   end
 
   def populate_realms_and_validate
     begin
       populate_realms
-    rescue
+    rescue => e
       errors.add(:base, I18n.t("provider_accounts.errors.populate_realms_failed", :message => $!.message))
```
Comment (jguiditta) [inline src/app/models/provider_account.rb:181]: just to be consistent, should this not also s/$!/e?

File: src/app/models/provider_account.rb [hunk 2]
```diff
@@ -52,9 +56,11 @@ class << self
            :include => [:role],
            :order => "derived_permissions.id ASC"
 
-  # validation of credentials is done in provider_account validation, :validate => false prevents nested_attributes from validation
+  # Calidation of credentials is done in provider_account validation,
```
Comment (jguiditta) [inline src/app/models/provider_account.rb:59]: Github doesn't seem to show it, but there is trailing whitespace on this line, and typo 'Calidation'

File: src/app/models/provider_account.rb [hunk 3]
```diff
@@ -128,14 +136,16 @@ def destroyable?
 
   def connect
     begin
-      opts = {:username => credentials_hash['username'],
-              :password => credentials_hash['password'],
-              :driver => provider.provider_type.deltacloud_driver }
+      opts = {
+        :username => credentials_hash['username'],
+        :password => credentials_hash['password'],
+        :driver => provider.provider_type.deltacloud_driver
```
Comment (jguiditta) [inline src/app/models/provider_account.rb:142]: trailing whitespace

File: src/app/models/provider_account.rb [hunk 4]
```diff
@@ -358,16 +358,23 @@ def instance_matches(instance, matched, errors)
...
+              :frontend_realm_name => instance.frontend_realm.name
+            )
             next
           end
           brealms.each do |brealm_target|
             # add match if realm is mapped to provider or if it's mapped to
             # backend realm which is available and is accessible for this
             # provider account
-            if (brealm_target.target_realm.nil? || (brealm_target.target_realm.available && realms.include?(brealm_target.target_realm)))
+            if (brealm_target.target_realm.nil? ||
```
Comment (jguiditta) [inline src/app/models/provider_account.rb:376]: trailing whitespace

**Comment by jguiditta** [comment]
Unfortunately, in addition to the minor notes I made as I was reading through your changes, this patch adds 59 new failures to the spec suite and 21 to the cucumber suite, so unfortunately this is not currently in an ACKable state.

**Comment by mfojtik** [comment]
Thanks Jay I will try to fix those failures asap. Need to setup som testing env somewhere to do so :)

**Pull request closed without merging**